trade-finance-manager-api/src/graphql/resolvers/query-facilities.js

```diff
@@ -8,17 +10,42 @@ exports.queryAllFacilities = async (queryParams) => {
   const facilities = [];
```

Comment:
I would have added some validation for `queryParams` to ensure its not empty, however not always required if confident the calling function has already been vetted. If not then please add some validation before executing expensive API calls.

Reply:
done